esp-hal/src/dma/gdma.rs

```diff
@@ -285,21 +310,36 @@ pub struct ChannelRxImpl<C: GdmaChannel>(C);
 impl<C: GdmaChannel> crate::private::Sealed for ChannelRxImpl<C> {}
 
 impl<C: GdmaChannel> ChannelRxImpl<C> {
+    #[cfg(not(esp32p4))]
```

Comment:
TBH I'd prefer not touching DMA for the P4.

Reply:
I think there's nothing critical to have minor changes in terms of registers/fields management there, I think it can stay there